proto/tendermint/abci/types.proto

```diff
@@ -355,7 +367,7 @@ message ResponseFinalizeBlock {
   repeated Event events = 1
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   repeated ExecTxResult            tx_results              = 2;
-  repeated ValidatorUpdate         validator_updates       = 3;
+  repeated ValidatorUpdate         validator_updates       = 3 [(gogoproto.nullable) = false];
```

Comment:
This now matches the `.intermediate` file. It appears that at some point these pieces of data started to skew.